tests/runtime/db/test_db.py

```diff
@@ -17,10 +16,11 @@ def test_default(self):
         # Then
         assert location == Path.home() / ".orquestra" / "workflows.db"
 
-    def test_with_environment_variable(self, tmp_path, monkeypatch):
+    def test_with_environment_variable(
+        self, tmp_path: Path, monkeypatch: pytest.MonkeyPatch
+    ):
         # Given
-        env = {"ORQ_DB_LOCATION": str(tmp_path / "workflows.db")}
-        monkeypatch.setattr("os.environ", env)
+        monkeypatch.setenv("ORQ_DB_PATH", str(tmp_path / "workflows.db"))
```

Comment:
**suggestion**: wanna use the constant from the `_env` module?

Reply:
I didn't want to use the stuff inside the package so that any changes in the file will cause tests to fail.
If we use that module inside the tests, changes will not cause a failure and we might break things for users.